### ISSUE TITLE
Default port of zabbix-web-* changed from 80 to 8080

### ISSUE THREAD

**Issue by kees24** [zabbix/zabbix-docker#698]
Dear All,

First, thanks for making a docker image, it works perfect and is easy to maintain, upgrade and expand. Life is so much easier.

During a upgrade from zabbix docker 4.4-latest to 5.0-latest I had trouble with the zabbix frontend. I was unable to access it without clear reason. The proxies and server were working but the frontend wasn't, logs showed everything was fine but still no webpage.

After some trouble shooting I found out that the port now is 8080 instead of just 80. I'm not a git guru so I'm unable to find out when it has changed. I coudn't find anything about it on the docker page (https://hub.docker.com/r/zabbix/zabbix-web-apache-mysql). I'm using MacVLAN so it exposes the ports that are defined by the container, so no portmapping but even for remapping it's important to know, right?

My question, is this accepted behavior? Why has it changed? Should we update the documentation? Or can we change it back / make it configurable?

Thanks in advance.

**Comment by kees24** [comment]
Thanks, my bad. I spotted them to late.

My questions are still valid I think. Can we change it back (or make it configurable) and updating the documentation?

I would like to hear a reason for the change and keeping it this way? Also with keeping ipv6 in mind where portmapping isn't existing (as far as I know last time I looked)

Thanks

(the other issues are closed so do we continue this one or reopen a old one?)
